packages/openactive-integration-tests/test/features/details-capture/customer-details-capture-identifier/implemented/customer-identifier-capture-test.js

```diff
@@ -20,62 +14,23 @@ FeatureHelper.describeFeature(module, {
   // The secondary opportunity criteria to use for multiple OrderItem tests
   controlOpportunityCriteria: 'TestOpportunityBookable',
 },
-function (configuration, orderItemCriteria, featureIsImplemented, logger, state, flow) {
-  describe('Customer identifier reflected back at C2 and B', () => {
-    beforeAll(async function () {
-      await state.fetchOpportunities(orderItemCriteria);
-      return chakram.wait();
-    });
-
-    afterAll(async function () {
-      await state.deleteOrder();
-      return chakram.wait();
-    });
-
-    describe('Get Opportunity Feed Items', () => {
-      (new GetMatch({
-        state, flow, logger, orderItemCriteria,
-      }))
-        .beforeSetup()
-        .successChecks()
-        .validationTests();
-    });
-
-    describe('C1', () => {
-      (new C1({
-        state, flow, logger,
-      }))
-        .beforeSetup()
-        .successChecks()
-        .validationTests();
+function (configuration, orderItemCriteriaList, featureIsImplemented, logger) {
+  // # Initialise Flow Stages
+  const { fetchOpportunities, c1, c2, b } = FlowStageRecipes.initialiseSimpleC1C2BFlow(orderItemCriteriaList, logger);
+
+  // # Set up Tests
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(fetchOpportunities);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(c1);
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(c2, () => {
+    it('should include expected Customer identifier', () => {
+      // CustomerIdentifierC2 is set in the standard C2 request template.
+      expect(c2.getOutput().httpResponse.body.customer.identifier).to.equal('CustomerIdentifierC2');
     });
-
-    describe('C2', function () {
-      (new C2({
-        state, flow, logger,
-      }))
-        .beforeSetup()
-        .successChecks()
-        .validationTests();
-
-      it('should return 200, with an Expected customer identifier', () => {
-        chai.expect(state.c2Response.response.statusCode).to.equal(200);
-        chai.expect(state.c2Response.body.customer.identifier).to.equal('CustomerIdentifierC2');
-      });
-    });
-
-    describe('B', function () {
-      (new B({
-        state, flow, logger,
-      }))
-        .beforeSetup()
-        .itResponseReceived()
-        .validationTests();
-
-      it('should return 200, with an Expected customer identifier', () => {
-        chai.expect(state.bResponse.response.statusCode).to.equal(200);
-        chai.expect(state.bResponse.body.customer.identifier).to.equal('CustomerIdentifierB');
-      });
+  });
+  FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(b, () => {
+    it('should include expected Customer identifier', () => {
+      // CustomerIdentifierB is set in the standard B request template.
+      expect(b.getOutput().httpResponse.body.customer.identifier).to.equal('CustomerIdentifierB');
```

Comment:
I raised an issue about this btw: https://github.com/openactive/openactive-test-suite/issues/361

the fact that B uses a different customer to C2 in the standard request templates.

In this PR, I haven't changed this